app/helpers/hosts_and_hostgroups_helper.rb

```diff
@@ -46,28 +46,30 @@ def os_ptable
     @operatingsystem.ptables
   end
 
-  def puppet_master_fields(f)
-    "#{puppet_ca(f)} #{puppet_master(f)}".html_safe
+  def puppet_master_fields(f, can_override = false)
+    "#{puppet_ca(f, can_override)} #{puppet_master(f, can_override)}".html_safe
   end
 
-  def puppet_ca(f)
+  INHERIT_TEXT = _("inherit")
```

Comment:
This is evaluated in the class context, so won't be translated to the user's locale.